hack/release/go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/mesosphere/kommander-applications/hack/release
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.4
```

Comment:
remove this

Reply:
@cwyl02 , The build was failing with the previous version, Do I have to still remove the changes. Please suggest.

Reply:
@cwyl02 I have reverted the changes , the build is failing in **go test**. suggest on if the go mod should be updated or not.